src/app/modals/search-modal/search-modal.component.ts

```diff
@@ -248,6 +248,8 @@ export class SearchModal implements OnInit, AfterViewInit {
       }
     }
 
+    queryParams['advSearch'] = true;
```

Comment:
Is this a bug?

Reply:
no.
it adds "advSearch" as a query param.
that is referenced in the asset-grid
`this.route.snapshot.params.advSearch`

Reply:
Oh I see, I thought we can already determine the type of search. Reviewing it to see if I can find it.

Reply:
I could find a way.
but it's been some time since i've angular(ed). so maybe there's some magic i don't know about 😅

Reply:
I was thinking of this, but it checks if a specific filter was applied, not any: https://github.com/ithaka/aiw-ui/blob/01265f0bf037d087a0f139abe227f33cb6b1e5a5/src/app/asset-filters/asset-filters.service.ts#L180-L194

Reply:
🤔  I don't think this service gives me anything that tells me it was an advanced search or not